### ISSUE TITLE
Unable to deploy snapshots to Maven (presumably since switch to 8.1)

### ISSUE THREAD

**Issue by pihme** [camunda/zeebe#9087]
**Description**

Since this week we are unable to deploy snapshots to Maven. The upload step hangs with the following error message:
```
[2022-04-11T09:55:13.579Z] [INFO] Waiting for other projects build to finish...
[2022-04-11T09:55:15.477Z] [INFO] Waiting for other projects build to finish...
[2022-04-11T09:55:17.373Z] [INFO] Waiting for other projects build to finish...
[2022-04-11T09:55:19.270Z] [INFO] Waiting for other projects build to finish...
[2022-04-11T09:55:21.794Z] [INFO] Waiting for other projects build to finish...
[2022-04-11T09:55:23.691Z] [INFO] Waiting for other projects build to finish...
[2022-04-11T09:55:24.505Z] Sending interrupt signal to process
[2022-04-11T09:55:24.505Z] Killing processes
[2022-04-11T09:55:24.830Z] kill finished with exit code 0
```

There is this bug which sounds similar in terms of symptoms https://issues.sonatype.org/browse/NEXUS-6735

However, AFAIK we are not using the config property mentioned and we don't skip modules.

**Comment by pihme** [comment]
Slack communication: https://camunda.slack.com/archives/CSQ2E3BT4/p1649670981498129

**Comment by npepinpe** [comment]
The commit which triggered this ([here](7e0c555a1a33ae749a319408f3bfd8f3592cb1aa)) removed a module. I'm wondering if this isn't the problem. But then, is there no way to remove modules in Maven?

**Comment by npepinpe** [comment]
It looks like the `zeebe-protocol` is the module which starts being deployed but never gets deployed. My guess is the generation of the Javadoc is an issue, or the source JAR, not sure which one, but these are steps we rarely do except during deployment.

**Comment by npepinpe** [comment]
Yup, there's an error when generating the Javadoc for the protocol module starting at this commit. I'm curious why it fails the deployment but no error is printed out. Should we start generating (or if possible only validating) the Javadoc in our build phase as well?
